src/Protocol/ActivityPub/Receiver.php

```diff
@@ -1257,17 +1287,23 @@ private static function processAttachmentUrls(array $object, array $object_data)
 				continue;
 			}
 
-			if ($mediatype == 'text/html') {
-				$object_data['alternate-url'] = $href;
-			}
-
 			$filetype = strtolower(substr($mediatype, 0, strpos($mediatype, '/')));
 
 			if ($filetype == 'audio') {
 				$attachments[$filetype] = ['type' => $mediatype, 'url' => $href, 'height' => null, 'size' => null];
 			} elseif ($filetype == 'video') {
 				$height = (int)JsonLD::fetchElement($url, 'as:height', '@value');
+				// PeerTube audio-only track
```

Comment:
I think we should handle them as well. The first goal should be to store all data. Later (when processing the content) we can decide what to display.

Reply:
The problem is that the filetype is considered video, but it doesn't have a video track. Should I just put `'type' => 'audio'` for these?

Reply:
We could check the mimetype. Somewhere we have got a function for this.

Reply:
The mimetype is `video/mp4`, no salvation on this side.

Reply:
What happens when it is embedded in video tags?

Reply:
It plays sound but the video is blank even though it takes as much space as a regular video embed.

Reply:
Sounds like a valid thing.